packages/medusa/src/models/return-item.ts

```diff
@@ -19,6 +19,13 @@ import {
 import { Return } from "./return"
 import { LineItem } from "./line-item"
 
+export enum ReturnReason {
+  TOO_SMALL = "too_small",
+  TOO_BIG = "too_big",
```

Comment:
I think these return reasons are a bit too specific. Would rather have the predefined reasons to be more general, so we can fit more into the box e.g. `wrong_size` for the two size reasons here. Let me know, what you think.

Reply:
I think it brings a lot of value to the store operator to be able to tell in which direction their sizing is off, in order to make adjustments/recommendations. That being said I agree that these reasons are too specific - I think `wrong_size` is even too specific, considering the fact that products might not even have a size dimension. 

The proper way is probably to give store operators the option of defining their own return reasons. 

Will do that instead :)